sqlstore/migrations/0001_initial.sql

```diff
@@ -25,7 +25,7 @@ create table assets
     symbol              TEXT NOT NULL,
     total_supply        HUGEINT,
     decimals            INT,
-    quantum             INT,
+    quantum             HUGEINT,
```

Comment:
Quantum is actually a float not a integer.

Reply:
(chatted about this with Jeremey and resolved that while it is a num.Decimal in core, there is never any floating part point)